addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HubStatusListener.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.harmonyhub.handler;
+
+import org.eclipse.smarthome.core.thing.ThingStatus;
+
+/**
+ * the {@link HarmonyDeviceHandler} interface is for classes wishing to register
+ * to be called back when a HarmonyHub status changes
+ *
+ * @author Dan Cunningham - Initial contribution
+ *
+ */
+public interface HubStatusListener {
```

Comment:
I don't need it for the child thing, but the discovery class still needs to know when the hub goes online in order to scan for new devices ( https://github.com/digitaldan/openhab2-addons/blob/harmonyhub/addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/discovery/HarmonyDeviceDiscoveryService.java )  If you know of a better way of triggering the the discovery class without this I could remove the interface and logic.